packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -142,11 +142,25 @@ class Directionality extends InheritedWidget {
 /// This is more efficient than adding and removing the child widget from the
 /// tree on demand.
 ///
+/// ## Opacity Animation
+///
+/// `Opacity` animations should be built using [AnimatedOpacity] rather than
+/// manually updating the `Opacity` Widget.
```

Comment:
[Opacity] widget

Reply:
Maybe 'recreating' instead of 'updating'. Strictly speaking, immutable widgets aren't updatable.